apps/ponder-interop/src/createPonderConfig.ts

```diff
@@ -36,6 +39,16 @@ export function createPonderConfig(endpoints: Endpoints) {
         ]),
       ),
     },
+    GasTank: {
```

Comment:
Lets omit this field if the env variable is empty?

Reply:
yeah, that was the path I initially went down, but ponder types do not play well with the conditional GasTank. Let me check with the ponder team on whether this is supported

Reply:
havent heard back from ponder team, but this works as-is so going to merge for now and follow up if their recommended approach is different